<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>531</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2" columnstretch="0">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QGridLayout" name="centralGridLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item row="1" column="0" alignment="Qt::AlignTop">
       <widget class="QWidget" name="editWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="editLayout">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>704</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionGoToTask"/>
   <addaction name="actionGoToCal"/>
   <addaction name="actionGoToResume"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAddTask">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="test" resource="ressources.qrc">
     <normaloff>:/icons/img/icons/ic_add_to_photos_48px.svg</normaloff>:/icons/img/icons/ic_add_to_photos_48px.svg</iconset>
   </property>
   <property name="text">
    <string>AddTask</string>
   </property>
   <property name="iconText">
    <string>Ajouter tâche</string>
   </property>
   <property name="toolTip">
    <string>Ajouter une tâche</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="priority">
    <enum>QAction::HighPriority</enum>
   </property>
  </action>
  <action name="actionGoToTask">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/img/icons/ic_assignment_48px.svg</normaloff>:/icons/img/icons/ic_assignment_48px.svg</iconset>
   </property>
   <property name="text">
    <string>Tâches</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionGoToCal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/img/icons/ic_today_48px.svg</normaloff>:/icons/img/icons/ic_today_48px.svg</iconset>
   </property>
   <property name="text">
    <string>Calendrier</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionGoToResume">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/img/icons/ic_account_balance_48px.svg</normaloff>:/icons/img/icons/ic_account_balance_48px.svg</iconset>
   </property>
   <property name="text">
    <string>Résumé</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
